### PR TITLE
update sidecars to latest versions available

### PR DESCRIPTION
**Pull request by humblec**
Below sidecars are updated:

csi-provisioner: v3.3.0
csi-snapshotter: v6.1.0

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Review by Rakshith-R — CHANGES_REQUESTED** [review]
https://github.com/ceph/ceph-csi/blob/1eaf6af8824b2367ef07ee1e296f319d72bcaa86/build.env#L53

Changes required in build env too?

**Review by Rakshith-R — DISMISSED** [review]

**Review by yati1998 — DISMISSED** [review]

File: charts/ceph-csi-cephfs/values.yaml [hunk 1]
```diff
@@ -171,7 +171,7 @@ provisioner:
   provisioner:
     image:
       repository: registry.k8s.io/sig-storage/csi-provisioner
-      tag: v3.2.1
+      tag: v3.3.0
```
Comment (nixpanic) [inline charts/ceph-csi-cephfs/values.yaml:174]: in the commit message you mention version 3.3.1. Which is the correct one?

**Review by nixpanic — CHANGES_REQUESTED** [review]

File: charts/ceph-csi-cephfs/values.yaml (hunk 1 above)
Comment (Madhu-1) [inline charts/ceph-csi-cephfs/values.yaml:174]: Update in helm documentation also

**Review by Madhu-1 — DISMISSED** [review]

**Comment by nixpanic** [comment]
the 2 commits can be combined into one, no need to split this out, both are just rebasing the version for the same components

**Comment by humblec** [comment]
> the 2 commits can be combined into one, no need to split this out, both are just rebasing the version for the same components

@nixpanic done.. squashed the commits.

File: charts/ceph-csi-cephfs/values.yaml (hunk 1 above)
Comment (humblec) [reply charts/ceph-csi-cephfs/values.yaml:174]: Done

Comment (humblec) [reply charts/ceph-csi-cephfs/values.yaml:174]: 3.3.0 is the correct one. corrected the commit message

**Review by nixpanic — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Pull request merged**